src/main/java/edu/hm/hafner/analysis/parser/MsBuildParser.java

```diff
@@ -48,6 +49,16 @@ protected Optional<Issue> createIssue(final Matcher matcher, final LookaheadStre
 
         builder.setFileName(fileName);
 
+        if (StringUtils.isNotEmpty(matcher.group("severity"))){
```

Comment:
Hey @uhafner 

I added the code to check the group in "severity".

I also used the debugger. But my code does not goes into my if condition.

Instead it always goes to this line https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/parser/MsBuildParser.java#L81.

Am I using the group in a wrong way? Could you share a link to understand how to use it? I did not find much examples online. They were basic compared to our regex here.